apkenv.c

```diff
@@ -227,6 +243,82 @@ install_overrides(struct SupportModule *module)
     }
 }
 
+void
+dalvikvm_JNIEnv_DeleteLocalRef(JNIEnv* p0, jobject p1)
+{
+#ifdef APKENV_DEBUG
+    printf("dalvikvm_JNIEnv_DeleteLocalRef(%x)\n", p1);
+#endif
+    global.dalvik_copy_env.DeleteLocalRef(p0, p1);
+    (*p0)->ExceptionClear(p0);
+}
+
+
+static void
+install_default_dalvik_overrides()
+{
+    struct JNINativeInterface *dalvik_env = (struct JNINativeInterface *)global._env->functions;
+
+    int pagesize = sysconf(_SC_PAGESIZE);
+    mprotect((char*)((int)dalvik_env & ~(pagesize-1)), pagesize*2, PROT_READ | PROT_WRITE);
+
+    dalvik_env->NewGlobalRef = JNIEnv_NewGlobalRef;
+    dalvik_env->DeleteGlobalRef = JNIEnv_DeleteGlobalRef;
+
+    dalvik_env->GetObjectClass = JNIEnv_GetObjectClass;
+
+    dalvik_env->FindClass = JNIEnv_FindClass;
```

Comment:
Are all those overrides still necessary, or could we just use DVM's implementation of e.g. `GetFieldID`?